src/Dropdown/DropdownMenu.tsx

```diff
@@ -12,29 +13,43 @@ const directionPositionMap = {
 	down: "bottom"
 };
 
-const DropdownMenu = (props, context) => {
+export interface DropdownMenuProps {
+	/** Customize component output by passing in an element name or Component */
+	tag?: React.ComponentType | string;
+	/** Placement */
+	right?: boolean;
+	/** Modifier used to flip the popper’s placement when it starts to overlap its reference element. */
+	flip?: boolean;
+	/** Plugins used to alter the popper behavior */
+	modifiers?: Modifiers;
+	/** Class name */
+	className?: string;
+	/** Should menu persist */
+	persist?: boolean;
+}
+
+const DropdownMenu: React.FC<DropdownMenuProps> = (props, context) => {
 	const { className, right: rightPlacement, tag, flip, modifiers, persist, ...attrs } = props;
 	const classes = classNames(className, "dropdown-menu", {
 		"dropdown-menu-right": rightPlacement,
 		show: context.isOpen
 	});
 
-	const Tag = tag;
+	const Tag = tag as any;
```

Comment:
Having comment above fixed could be get rid of 'as any'?

Reply:
Nope 
![image](https://user-images.githubusercontent.com/13631794/68278243-96d7f680-0079-11ea-97fa-05da325b8eaf.png)

Reply:
Did you specify the type of `tag` prop as `ElementType`?

Reply:
For some reason this worked for me:
`const { className, right: rightPlacement, tag: Tag = "ul", flip, modifiers, persist, ...attrs } = props;`

Reply:
Yeah, it cannot be undefined with default value.